alacritty.yml

```diff
@@ -450,6 +468,27 @@
     # binding section.
     #modifiers: None
 
+# Regex hints
+#
+# Terminal hints can be used to find text in the visible part of the terminal
+# and pipe it to other applications.
+#hints:
+  # Keys used for the hint labels
+  #alphabet: "jfkdls;ahgurieowpq"
+
+  # List with all available hints
+  #
+  # The fields `command`, `binding.key` and `binding.mods` accept the same
+  # values as they do in the `key_bindings` section.
+  #
+  # Example:
+  #  - regex: "alacritty/alacritty#\\d*"
```

Comment:
This should really also be a list. I imagine people using the `open` command for a set of regexs that aren't trivial to unify.

Reply:
It shouldn't be a problem to combine multiple regexes into one?

Reply:
I mean, I suppose you're right. I just feel like it's going to turn into a pretty gross string (where whitespace matters), whereas an array would work well too? Perhaps this is a small issue.